core/templates/dev/head/css/oppia.css

```diff
@@ -2565,6 +2565,10 @@ div#ng-curtain {
   padding: 17px 12px 10px 12px;
   text-transform: uppercase;
 }
+.nav-block {
```

Comment:
I was thinking of it as a more general use class, in case of any additional (I didn't see any in a quick look though) or future uses.

I hadn't sent the reply to the email yet, but my thoughts were that removing .nav might not be worth it since it's so simple to apply the previous behavior and removing it would mean essentially copying and renaming several classes from bootstrap.

What would a more appropriate/scoped name be? Something like .editor-answer-nav?